lib/erlef_web/templates/blog/form.html.eex

```diff
@@ -0,0 +1,164 @@
+<div class="card card-primary">
+  <%= form_for @changeset, @action, [multipart: true], fn f -> %>
+    <div class="card-body">
+      <%= if @changeset.action do %>
+        <div class="alert alert-danger">
+          <p>Oops, something went wrong! Please check the errors below.</p>
+        </div>
+      <% end %>
+
+      <div class="form-group">
+        <%= label f, :title %>
+        <%= text_input f, :title, class: "form-control form-control-lg", oninput: "save(this)"%>
+        <%= error_tag f, :title %>
+        <%= error_tag f, :slug %>
+      </div>
+
+      <div class="form-group">
+        <%= label f, :authors %>
+        <div class="input-group flex-row">
+          <%= text_input f, :authors, class: "post-form-authors", value: authors_value(f), onchange: "save(this)"%>
```

Comment:
We might want to add an outline / border to this field so it's more clear that it's a field, especially in regards to error messages, what do you think?